cookiecutter.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Instead of the top level, can we put the cookie cutter configuration in something like *Utilities/CookieCutter*, e.g.?

Reply:
Done in 5409bf6.